app/IOSGateway.m

```diff
@@ -0,0 +1,121 @@
+//
+//  IOSGateway.m
+//  iSH
+//
+//  Created by Miguel Vanhove on 15/02/2019.
+//
+
+#import "IOSGateway.h"
+#import "IACManager.h"
+#import "UIApplication+OpenURL.h"
+
+static IOSGateway *iOSGateway = nil;
+
+@interface IOSGateway ()
```

Comment:
Is there a reason you're defining the interface for `IOSGateway` twice?

Reply:
It's to add the "NSData *iacResult" as private method